table/block_fetcher.cc

```diff
@@ -233,6 +238,7 @@ inline void BlockFetcher::GetBlockContents() {
 }
 
 IOStatus BlockFetcher::ReadBlockContents() {
+  std::vector<FSReadRequest> read_reqs;
```

Comment:
Do we need a `std::vector`? Looks like we always call `MultiRead` with here with one read request.

Reply:
No we don't. I'll update the code and remove `std::vector` and use single `FSReadRequest`